internal/empty-free/empty_free.go

```diff
@@ -3,9 +3,10 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-// Build when CGO is disabled
-//go:build !cgo
+package empty_free
 
-package waf
+import "unsafe"
 
-var disabledReason = "cgo was disabled during the compilation and should be enabled in order to compile with the waf"
+//go:linkname _empty_free _empty_free
+var _empty_free byte
+var EmptyFreeFn uintptr = uintptr(unsafe.Pointer(&_empty_free))
```

Comment:
Let's rename it `NoopFreeFn`:
```suggestion
var NoopFreeFn uintptr = uintptr(unsafe.Pointer(&_empty_free))
```